reflex-algos/src/main/python/mlcomp/Makefile

```diff
@@ -1,12 +1,34 @@
 # It make sense also to move all the component documentation to be done via python sphinx
 
+MLCOMPDISTDIR  = ""
+
+MLCOMP_EGG_PY2_SYMLINK = "ml_comp-py2.egg"
+MLCOMP_EGG_PY3_SYMLINK = "ml_comp-py3.egg"
+
 bdist:
 	python setup.py bdist
 
 egg:
+	rm -rf ./dist
 	python2 setup.py -q bdist_egg
 	python3 setup.py -q bdist_egg
 
+MLCOMP_EGG_PY2 = $(shell realpath $(shell ls ./dist/*-py2.*.egg))
+MLCOMP_EGG_PY3 = $(shell realpath $(shell ls ./dist/*-py3.*.egg))
+MLCOMP_EGG_PY2_BASENAME = $(shell basename $(MLCOMP_EGG_PY2))
+MLCOMP_EGG_PY3_BASENAME = $(shell basename $(MLCOMP_EGG_PY3))
+
+install_to: egg
```

Comment:
If you put here the `egg` dependency, you can rely on the fact that the eggs will be created under the `dist` folder. So, you can just copy them instead of generating them again with `MLCOMPDISTDIR`.

Reply:
fixed